### DISCUSSION TITLE
[建议] 适当增大表情包的大小

### DISCUSSION THREAD

**Discussion by hotarugali** [walinejs/waline#456] [Q&A] [answered]
目前 waline 显示的表情包大小和字体一样大，但这样看起来显得比较小，有些看不清楚。

![image](https://user-images.githubusercontent.com/32862896/128444504-76f537a0-45c6-4931-8566-2181c59f3d30.png)

建议能否适当增大一下表情包的大小，就和 valine 或 twkioo 这样的比例大小：
![image](https://user-images.githubusercontent.com/32862896/128445106-fd7d926f-0bf8-42cf-ba77-211e630a32c2.png)

**Comment by Mister-Hope** [comment, marked as answer]
你可以自由的自己调整css

**Comment by hotarugali** [comment]
好的，我研究一下，谢谢回复~
